src/TestUtils/src/Microsoft.Maui.IntegrationTests/TemplateTests.cs

```diff
@@ -181,6 +182,58 @@ public void BuildWithDifferentVersionNumber(string id, string config, string dis
 				$"Project {Path.GetFileName(projectFile)} failed to build. Check test output/attachments for errors.");
 		}
 
+		List<string> SearchForExpectedEntitlements(string entitlementsPath, string appLocation, List<string> expectedEntitlements)
```

Comment:
Total nit here, but could this be moved into a new file in the Apple folder? Maybe Apple/Codesign.cs so it can be used from more than template tests?

I am on a mission-ish to get more test coverage of all our scary targets, so having all the platform "tools" in a place outside the tests will be great for re-use.